public/js/popup.js

```diff
@@ -58,21 +83,30 @@ function warningModal(clone, options) {
         const errorCell = line.insertCell(2);
         errorCell.classList.add("highlight");
         errorCell.classList.add("msg");
+
         const positionCell = line.insertCell(3);
         positionCell.classList.add("position");
+        if (kind !== "short-identifiers" && kind !== "obfuscated-code") {
```

Comment:
```suggestion
        if (!file.includes(".min") && kind !== "short-identifiers" && kind !== "obfuscated-code") {
```

I don't think we should run the feature for minimized files.

Reply:
Obviously it will be pretty unreadable ^^